docs/experiments/categories/gcp/gcp-vm-disk-loss/gcp-disk-loss.yaml

```diff
@@ -0,0 +1,30 @@
+## details of the gcp disk
+apiVersion: litmuschaos.io/v1alpha1
+kind: ChaosEngine
+metadata:
+  name: engine-nginx
```

Comment:
Can we keep the name as `gcp-disk-chaos`, as that is what we have mentioned in the docs as default?